web/libs/storybook/src/assets/icons/index.ts

```diff
@@ -0,0 +1,3 @@
+/// <reference types="vite-plugin-svgr/client" />
+
```

Comment:
this is a good opportunity to clean things up
```suggestion
```

Reply:
without it for some reason TS complains about types (can't import things with `?react`). gotta figure out a different way. but this is recommended by the svgr themself

Reply:
interesting, then lets disregard that - thanks for following up!